recommend/recommend.go

```diff
@@ -106,6 +211,10 @@ func Recommend(c *k8s.Client, o Options) error {
 		return err
 	}
 
+	if o.Merge == "merge" {
```

Comment:
this will throw error if the user is making use of `karmor recommend` for the first time since there will be no folder to look for policies.

Reply:
thank you for your thorough review and Changes are updated.